bin/run-integration

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 if [[ "$RUN_ARGS" == "--nightly" ]];
 then
-    MOCHA_FILE=./test-results-integration.xml npm run test-integration -- -R mocha-junit-reporter -t $CIRCLE_NODE_TOTAL -i $CIRCLE_NODE_INDEX
+	npm run test-integration:ci
```

Comment:
I have it working for server tests using files option and glob. I didn't find proper setup for integration tests so far. I'm sure it's possible.